src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/UpdateSubmission.java

```diff
@@ -29,9 +29,6 @@ public class UpdateSubmission {
     public static final String CESSATIONS_FIELD = "cessations";
     public static final String TRUSTS_ADDITIONS_FIELD = "trusts";
 
-    @JsonProperty(UPDATE_TYPE_FIELD)
```

Comment:
should this be left in? is it used by filing-resource-handler or chips-filing-consumer?